grammars/jsdoc.cson

```diff
@@ -252,28 +275,29 @@
         'name': 'variable.other.jsdoc'
       }
       {
-        'name': 'variable.other.jsdoc'
         'begin': '\\['
-        'end': '\\]|(?=\\*/)'
+        'beginCaptures':
+          '0':
+            'name': 'variable.other.jsdoc'
+        'contentName': 'variable.other.jsdoc'
```

Comment:
In a theme which doesn't highlight `invalid.illegal.syntax.jsdoc`:

<img src="https://cloud.githubusercontent.com/assets/2346707/24089886/71edf0b6-0d8f-11e7-8777-c4eff968679a.png" width="378" alt="" />